Tasks/KubeloginInstallerV0/Tests/DownloadKubeloginReleaseL0Tests.ts

```diff
@@ -0,0 +1,34 @@
+import { getKubeloginRelease, KubeloginRelease, downloadKubeloginRelease} from "../utils";
+import { TestString } from "./TestStrings";
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import path = require('path');
+const fs = require('fs')
+
+
+const tempDir = path.join(__dirname, "temp");
+taskLib.setVariable("agent.TempDirectory", tempDir);
+taskLib.setVariable("Agent.ToolsDirectory", tempDir);
+
+export class DownloadKubeloginReleaseL0Tests {
+
+    public static async startTests() {
+        this.validateDownloadKubelogin();
+        if(!fs.existsSync(tempDir)) {
```

Comment:
Here and below - formatting: leave a space between if and opening bracket

Reply:
Yeah, Forgot about formatting. I will push with right format